libs/sdk-ui-kit/src/Dialog/ShareDialog/ShareDialogBase/GranularPermissions/GranularGranteeGroupItem.tsx

```diff
@@ -1,31 +1,46 @@
-// (C) 2022 GoodData Corporation
+// (C) 2022-2023 GoodData Corporation
```

Comment:
When we now have a special directory for granular stuff, we can as well have one for non-granular (simple?) to make it symmetrical

Reply:
To me it looks like the only thing that would make sense to separate at the same level are the components used inside `GranteeItem`, but they are all defined in the same file.

https://github.com/gooddata/gooddata-ui-sdk/blob/df0034209d81ba6c98116d68bd8dc97ea8a09b58/libs/sdk-ui-kit/src/Dialog/ShareDialog/ShareDialogBase/GranteeItem.tsx#L201-L205

Reply:
Keeping it as is.